integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -177,15 +186,19 @@ def write_documents(self, documents: List[Document], policy: DuplicatePolicy = D
     def delete_documents(self, document_ids: List[str]) -> None:
         """
         Deletes all documents with a matching document_ids from the document store.
+
         Fails with `MissingDocumentError` if no document with this id is present in the store.
```

Comment:
is this an exception being thrown? if so I would say added to the docstring with the `:raises:` keyword

Reply:
great catch - let me have a look

Reply:
dropped it